packages/build/src/plugins/child/validate.js

```diff
@@ -1,6 +1,6 @@
 import { addErrorInfo } from '../../error/info.js'
 import { serializeArray } from '../../log/serialize.js'
-import { EVENTS } from '../events.js'
+import { DEV_EVENTS, EVENTS } from '../events.js'
```

Comment:
I want to rename `EVENTS` to `BUILD_EVENTS` for consistency, but let's do it in a follow up because this PR is already too large.